package.json

```diff
@@ -48,7 +48,7 @@
     "typescript": "^4.7.4"
   },
   "peerDependencies": {
-    "graphql": "^0.12.0 || ^0.13.0 || ^14.0.0 || ^15.0.0"
+    "graphql": "^0.12.0 || ^0.13.0 || ^14.0.0 || ^15.0.0 || ^16.0.0"
```

Comment:
Since `graphql@15` -> `graphql@16` has breaking changes (e.g. the `graphql` function that we use [here](https://github.com/maticzav/graphql-middleware/pull/547/files#diff-1d69d3f7d438a8a52ef8dababf3b9bb08407334324bd026e72ce6038d8ea4b08R163) has a different signature), shouldn't we remove the versions `<16` from `peerDependencies`?

Reply:
Good point, I'm wondering how it will affect implementations, but the core functionality seems to be unaffected from what I can tell.